spec/junk_drawer/rails/bulk_updatable_spec.rb

```diff
@@ -83,6 +85,19 @@
         nested_jsonb_value: :integer,
         nested_jsonb_array_value: integer_array,
       )
+
+      class << self
+
+        attr_accessor :commit_count
+
+      end
+
+      belongs_to :related_model
+
+      def increment_commit_count
+        self.class.commit_count ||= 0
```

Comment:
Wondering if we should put this above inside the `class << self` bit, and that way our test looks less weird looking for `nil`:

```rb
class << self

  attr_writer :commit_count

  def commit_count
    @commit_count ||= 0
  end
end
```